src/js/node/stream.js

```diff
@@ -2282,9 +2282,13 @@ var require_readable = __commonJS({
         }
       });
     }
-    Readable.prototype = {};
-    ObjectSetPrototypeOf(Readable.prototype, Stream.prototype);
-    ObjectSetPrototypeOf(Readable, Stream);
+    // Set Readable's prototype to Stream's prototype,
+    // so instances of Readable will inherit from Stream
+    Readable.prototype = Stream.prototype;
```

Comment:
this isn't correct. i'll fix this on my pr

Reply:
yes please, when is it available?

Reply:
@paperdave any update pls

Reply:
pls @paperdave

Reply:
@paperdave